pyproject.toml

```diff
@@ -18,7 +18,7 @@ requires-python = ">=3.12"
 dependencies = [
     "argparse",
     "dotenv",
-    "jinja2>=3.1.6",
+    "jinja2",
```

Comment:
We stopped pinning dependencies a while ago.

Reply:
what do we even use jinja for?

Reply:
For injecting custom hosts into docker compose files, see https://github.com/dasch-swiss/dsp-tools/blob/39dbf00afd62e2a9f05b359e9eb7bdfddb49aa44/src/dsp_tools/commands/start_stack.py#L140

Reply:
ah, makes sense